src/main/java/dasniko/keycloak/authenticator/SmsAuthenticator.java

```diff
@@ -50,20 +53,22 @@ public void authenticate(AuthenticationFlowContext context) {
 			authSession.setAuthNote(CODE_TTL, Long.toString(System.currentTimeMillis() + (ttl * 1000L)));
 
 			String smsAuthText = theme.getMessages(locale).getProperty("authCodeText");
-			String smsText = String.format(smsAuthText, code, Math.floorDiv(ttl, 60));
+			Integer formattedTtl = Math.floorDiv(ttl, 60);
```

Comment:
took me a while to realize `ttl` comes in seconds. perhaps renaming the variable to a more descriptive name?